### PR TITLE
Supplement the admin stores configuration list with missing data

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describes:
- Security block
- Yotpo block
- Sales Channels block
-  Dotdigital block
in the admin stores configuration list.
## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/stores/configuration-overview.html

### CODE REVIEW

File: src/stores/configuration-overview.md [hunk 1]
```diff
@@ -25,6 +29,18 @@ _Configuration_
...
+
+[Yotpo]({% link configuration/yotpo.md %}) settings determine the Yotpo module, reviews, and visual marketing settings.
+
+### Sales Channels
+
+[Sales Chanel]({% link configuration/sales-channels.md %}) settings determine the clear log history, background tasks source, debag login, and read-only mode setting. Only available when Amazon Sales Channel is configured.
+
+### Dotdigital
+
+[Dotdigital]({% link configuration/dotdigital.md %}) settings determine the Dotdigital marketing solution setting. Allows set the build segments and triggers, data mapping, and creates a relevant account and marketing campaigns.
```
Comment (jcalcaben) [inline src/stores/configuration-overview.md:42]:
```suggestion
[Dotdigital]({% link configuration/dotdigital.md %}) settings configure the Dotdigital marketing feature. It configures the build segments and triggers, data mapping, and lets you create a relevant account and marketing campaigns.
```

**Review by jcalcaben — CHANGES_REQUESTED** [review]

**Comment by EvheniiaKolkhiieva** [comment]
Hi @jcalcaben, thank you for your review! 
I have implemented all suggestions you describe. Could you please review and approve this PR?

Thank you!

File: src/stores/configuration-overview.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/stores/configuration-overview.md]: Done ✅

File: src/stores/configuration-overview.md [hunk 2]
```diff
@@ -25,6 +29,18 @@ _Configuration_
 
 [Sales configuration]({% link configuration/sales.md %}) determines checkout and tax settings, payment and shipping options, sales email and PDF print-outs, and Google API settings.
 
+### Yotpo
+
+[Yotpo]({% link configuration/yotpo.md %}) settings determine the Yotpo module, reviews, and visual marketing settings.
+
+### Sales Channels
+
+[Sales Chanel]({% link configuration/sales-channels.md %}) settings determine the clear log history, background tasks source, debag login, and read-only mode setting. Only available when Amazon Sales Channel is configured.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/stores/configuration-overview.md]: Done ✅

File: src/stores/configuration-overview.md [hunk 3]
```diff
@@ -25,6 +29,18 @@ _Configuration_
 
 [Sales configuration]({% link configuration/sales.md %}) determines checkout and tax settings, payment and shipping options, sales email and PDF print-outs, and Google API settings.
 
+### Yotpo
+
+[Yotpo]({% link configuration/yotpo.md %}) settings determine the Yotpo module, reviews, and visual marketing settings.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/stores/configuration-overview.md]: Done ✅

File: src/stores/configuration-overview.md [hunk 4]
```diff
@@ -17,6 +17,10 @@ _Configuration_
 
 [Catalog configuration]({% link configuration/catalog.md %}) settings determine product and inventory settings, controls sitemap and RSS feed generation, and specifies the email template that is used to share products with friends.
 
+### Security
+
+[Security]({% link configuration/security.md %}) settings determine security settings, 2FA authentication and Google reCapture feature.
```
[the remark this answers is not in the document]
Comment (EvheniiaKolkhiieva) [reply src/stores/configuration-overview.md]: Done ✅

**Review by jcalcaben — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
@jcalcaben the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1227`.

**Pull request merged**